automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -109,30 +111,62 @@ fi
 function failure_check() {
     echo
     echo
-    if [[ $exitcode -ne 0 ]]; then
+    if [[ "${exitcode}" -ne 0 ]]; then
         echo "ERROR: UI test run failed, please check above URL"
     else
     echo "All UI test(s) have passed!"
     fi
 
     echo
     echo "COPY ARTIFACTS"
+    echo "RESULTS"
+    echo
     echo
-    cp -r ./results /builds/worker/artifacts
+    cp -r ./results "${ARTIFACT_DIR}"
+    ls -la "${RESULTS_DIR}" 
 
+    echo
     echo
     echo "RESULTS"
     echo
+    echo
     ls -la ./results
+
     echo
+    mkdir -p "${ARTIFACT_DIR}/github"
     echo
+    chmod +x "${PATH_TEST}/parse-ui-test.py"
+
+    "${PATH_TEST}/parse-ui-test.py" \
```

Comment:
Bug: For some reason, this line doesn't bail out when it fails. I see `set -e` is defined up above. I'm not sure what's wrong with it, but I believe we should fix it before it gets merge. This will prevent false positive like this task:

```
[task 2020-10-13T10:10:41.597Z] + chmod +x ./automation/taskcluster/androidTest/parse-ui-test.py
[task 2020-10-13T10:10:41.598Z] + ./automation/taskcluster/androidTest/parse-ui-test.py --exit-code 0 --log flank.log --results /builds/worker/artifacts/results --output-md /builds/worker/artifacts/github/customCheckRunText.md --device-type arm
[task 2020-10-13T10:10:41.623Z] Traceback (most recent call last):
[task 2020-10-13T10:10:41.623Z]   File "./automation/taskcluster/androidTest/parse-ui-test.py", line 7, in <module>
[task 2020-10-13T10:10:41.623Z]     import yaml
[task 2020-10-13T10:10:41.623Z] ModuleNotFoundError: No module named 'yaml'
[task 2020-10-13T10:10:41.626Z] + exit 0
[fetches 2020-10-13T10:10:41.626Z] removing /builds/worker/fetches
```

https://firefox-ci-tc.services.mozilla.com/tasks/XOXO6SmpSdiPfmsT22tZSA/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FXOXO6SmpSdiPfmsT22tZSA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log